atomata/capstone_WebClient
# User auth

### Summary 
using google to login.

### Details
In the login page, there is a google login button. Which will login with the googleId. it's already unique. 

### How to test?
Run and login with google account then try create. Log out, login again with google and see if it's still there. 

### Checks
